.github/pull_request_template.md

```diff
@@ -1,18 +1,17 @@
-You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md) and that you comply with the following criteria:
+ You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md).
 
-- [ ] You have sticked to Python. Please talk to us before adding other programming languages to HADDOCK3
-- [ ] Your PR is about CNS
-- [ ] Your code is well documented: proper docstrings and explanatory comments for those tricky parts
-- [ ] You structured the code into small functions as much as possible. You can use classes if there is a (state) purpose
-- [ ] Your code follows our coding style
-- [ ] You wrote tests for the new code
-- [ ] `tox` tests pass. *Run `tox` command inside the repository folder*
-- [ ] `-test.cfg` examples execute without errors. *Inside `examples/` run `python run_tests.py -b`*
-- [ ] PR does not add any dependencies, unless permission granted by the HADDOCK team
-- [ ] PR does not break licensing
-- [ ] Your PR is about writing documentation for already existing code :fire:
-- [ ] Your PR is about writing tests for already existing code :godmode:
+## Checklist
 
----
+- [ ] Tests added for the new code
```

Comment:
What about splitting uni and integration tests ?
```suggestion
- [ ] Uni-tests added for the new code (in `tests`)
- [ ] Integration-tests added for the new code (in `integration_tests`)
```

Reply:
Don't think the separation is relevant here